db-commons/src/main/scala/org/bitcoins/db/CRUD.scala

```diff
@@ -23,7 +23,7 @@ abstract class CRUD[T, PrimaryKeyType] extends BitcoinSLogger {
   def dbConfig: DbConfig
 
   /** Binding to the actual database itself, this is what is used to run querys */
-  def database: SafeDatabase = SafeDatabase(dbConfig)
+  def database: SafeDatabase[_] = SafeDatabase(dbConfig)
```

Comment:
Can you use `dbConfig.type` here? If not then I still think `[DbConfig]` is nicer than `[_]`